app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/types.ts

```diff
@@ -46,6 +49,15 @@ export interface ISelectColumnsProps {
   ) => void;
   assessmentLoading: boolean;
   tinkEnabled: boolean;
+  setTables: (
+    tables: ITablesStore,
+    columns: IColumnsStore,
+    dmlBlacklist: IDMLStore,
+    checkTransformations?: boolean
+  ) => void;
+  tables: ITablesStore;
+  columns: IColumnsStore;
```

Comment:
Could we make these namings more specific? like `tablesStore` and `columnsStore`